.eslintrc

```diff
@@ -158,6 +158,18 @@
     }
   },
   "overrides": [
+    {
+      "files": ["*.js", "*.jsx", "*.ts", "*.tsx"],
+      "rules": {
+        "no-unconditional-metabase-links-render": "error"
+      }
+    },
+    {
+      "files": ["*.unit.spec.*"],
```

Comment:
A lot of unit tests file contains literal metabase links in assertions, so I need to disable it.